packages/bpk-component-drawer/src/BpkDrawerContent-test.tsx

```diff
@@ -63,6 +63,22 @@ describe('BpkDrawerContent', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('should render correctly when it has padded=true', () => {
+    const { asFragment } = render(
+      <BpkDrawerContent
+        id="my-drawer"
+        title="Drawer title"
+        onClose={jest.fn()}
+        onCloseAnimationComplete={jest.fn()}
+        dialogRef={jest.fn()}
+        padded
```

Comment:
Test new prop via sub-component (this doesn't have the true default val which the Bpk Component has)